README.md

```diff
@@ -79,46 +56,30 @@ locally.
   `yourapp.herokuapp.com`. However, if you use a custom domain in production,
   that should be the value you use in production.
 - `JWT_SECRET_KEY`: generate with `rails secret`
-- `ELASTICSEARCH_INDEX`: Elasticsearch index or alias to query
-- `ELASTICSEARCH_URL`: defaults to `http://localhost:9200`
+- `OPENSEARCH_INDEX`: Opensearch index or alias to query
+- `OPENSEARCH_URL`: Opensearch URL
```

Comment:
I'm able to execute queries locally without a defined `OPENSEARCH_INDEX`, so I'm not sure that's required in all environments. As for `OPENSEARCH_URL`, if we are no longer defaulting to `localhost:9200`, I think we should add it to`.env.test`.

Reply:
Interesting. I'm curious what is being searched if not index is provided. I'll confirm that behavior and check whether these are required or optional. 👍🏻

Reply:
When no index is provided, it searches _all_ indexes. This is likely not a great idea so I updated the readme to suggest the more useful aliases.

I also moved these two to required in prod... which hopefully is interpreted as optional elsewhere?

Reply:
Thanks for adding that context. Yeah, I think it's an established pattern in our docs that specifying something as required in one environment implicitly specifies it as optional elsewhere.